CHANGELOG.md

```diff
@@ -71,6 +71,7 @@ Note that this project **does not** adhere to [Semantic Versioning](http://semve
 - We fixed the citation key generation for (`[authors]`, `[authshort]`, `[authorsAlpha]`, `authIniN`, `authEtAl`, `auth.etal`)[https://docs.jabref.org/setup/citationkeypatterns#special-field-markers] to handle `and others` properly. [koppor#626](https://github.com/koppor/jabref/issues/626)
 - We fixed the Save/save as file type shows BIBTEX_DB instead of "Bibtex library" [#9372](https://github.com/JabRef/jabref/issues/9372)
 - We fixed an issue regarding recording redundant prefixes in search history. [#9685](https://github.com/JabRef/jabref/issues/9685)
+- We fixed an issue where passing a valid URL containing a "?doi=" tag to DOI.parse yielded a "No entry found" notification. [#9821](https://github.com/JabRef/jabref/issues/9821)
```

Comment:
It's more than `?doi=`. Moreover, end users are not interested in implementation details. Here a suggestion:

```suggestion
- We fixed an issue where passing a URL containing a DOI lead to a "No entry found" notification. [#9821](https://github.com/JabRef/jabref/issues/9821)
```

Reply:
> Small comment. I assume you won't to a GUI test in this PR.

I'd certainly like to try. I assume I should follow this [comment](https://github.com/JabRef/jabref/issues/9821#issuecomment-1534166507) and in particular look at this [code](src/test/java/org/jabref/gui/search/GlobalSearchBarTest.java).